File: JS Advanced/Syntax, Functions, Statements/DaysOfWeek.js
let daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', "Thursday", 'Friday', 'Saturday', 'Sunday'];

function PrintPositionOfDay(day) {

    let index = daysOfWeek.indexOf(day);

    if (index == -1) {
        console.log('error');
        return;
    }

    console.log(index + 1);
}

File: C# Fundamentals/Arrays/DaysOfWeek.cs
using System;

namespace DaysOfTheWeek
{
    class Program
    {
        static void Main(string[] args)
        {
            var daysOfWeek = { "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" };
            var day = int.Parse(Console.ReadLine());

            if (day > 7 || day < 1)
            {
                Console.WriteLine("Invalid day!");
            }
            else
            {
                Console.WriteLine(daysOfWeek[day - 1]);
            }
        }
    }
}